### PR TITLE
Browser SDK v5 updates

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do?
- Adds instructions on what to do when upgrading from v4 to v5
- Updates variables across the various pages that have changed

This PR should be merged together with https://github.com/DataDog/documentation/pull/19745/files

### Motivation
[DOCS-5768](https://datadoghq.atlassian.net/browse/DOCS-5768)

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-5768]: https://datadoghq.atlassian.net/browse/DOCS-5768?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/real_user_monitoring/guide/browser-sdk-upgrade.md [hunk 1]
```diff
@@ -10,9 +10,195 @@ further_reading:
   text: "Use Datadog Session Replay to view real-time user journeys"
 ---
 
-## Overview 
+## Overview
 
-Follow this guide to migrate between major versions of the RUM and Logs Browser SDKs.
+Follow this guide to migrate between major versions of the Browser RUM and Logs Browser SDKs.
```
Comment (bcaudan) [inline content/en/real_user_monitoring/guide/browser-sdk-upgrade.md:15]:
```suggestion
Follow this guide to migrate between major versions of the Browser RUM and Browser Logs SDKs.
```

**Review by jhgilbert — APPROVED** [review]

File: content/en/real_user_monitoring/guide/browser-sdk-upgrade.md [hunk 2]
```diff
@@ -10,9 +10,193 @@ further_reading:
   text: "Use Datadog Session Replay to view real-time user journeys"
 ---
 
-## Overview 
+## Overview
 
-Follow this guide to migrate between major versions of the RUM and Logs Browser SDKs.
+Follow this guide to migrate between major versions of the Browser RUM and Browser Logs SDKs.
+
```
Comment (nhientrandatadog) [inline content/en/real_user_monitoring/guide/browser-sdk-upgrade.md:16]: Let's add something to point people to the latest SDK version, something like:

For the latest browser SDK capabilities, go [here](https://docs.datadoghq.com/real_user_monitoring/browser/).

File: content/en/real_user_monitoring/guide/browser-sdk-upgrade.md [hunk 3]
```diff
@@ -10,9 +10,193 @@ further_reading:
...
+## From v4 to v5
+
```
Comment (nhientrandatadog) [inline content/en/real_user_monitoring/guide/browser-sdk-upgrade.md:18]: Let's add the availability date here:

V5 will be available starting October XX, 2023.

File: content/en/real_user_monitoring/guide/browser-sdk-upgrade.md [hunk 4]
```diff
@@ -122,6 +122,12 @@ In v5, the default `sessionReplaySampleRate` is 0 instead of 100. If you don't i
 
 ### RUM
 
+### APM integration
+
+To promote the support and usage of OpenTelemetry, the default propagator types has been changed to include `tracecontext` in addition to `datadog`.
+
+**Action to take**: If you are not already specifying the desired propagator on the `allowedTracingUrls` initialization parameter, configure you server Access-Control-Allow-Headers to also accept `traceparent` header. For more information, see [connect RUM and Traces][25].
```
Comment (nhientrandatadog) [inline content/en/real_user_monitoring/guide/browser-sdk-upgrade.md:129]: small typo -- it should be "your* server"

File: content/en/real_user_monitoring/guide/browser-sdk-upgrade.md [hunk 5]
```diff
@@ -122,6 +122,12 @@ In v5, the default `sessionReplaySampleRate` is 0 instead of 100. If you don't i
 
 ### RUM
 
+### APM integration
+
+To promote the support and usage of OpenTelemetry, the default propagator types has been changed to include `tracecontext` in addition to `datadog`.
```
Comment (nhientrandatadog) [inline content/en/real_user_monitoring/guide/browser-sdk-upgrade.md:127]: should it be "default propagator types have*" or "default propagator type* has" instead? so the pluralization is in agreement

**Pull request merged**